packages/amplify-graphql-api-construct-tests/package.json

```diff
@@ -21,9 +21,11 @@
     "build-tests": "tsc --build tsconfig.tests.json"
   },
   "dependencies": {
+    "@aws-sdk/client-lambda": "^3.438.0",
```

Comment:
This is pulling in several additional dependencies with it which could eventually cause the CLI size to increase. I see that we already use `3.338.0` version for the SDK clients including the lambda client. Can we use that version instead?

Reply:
This shouldn't affect library size, this is is just test package dependencies, I can flip to use 338 though.